### PR TITLE
fix: rename gitea secret for better description

### LINKED ISSUE #216: The secret returned by the command "idpbuilder get secrets" should also include the username for argocd
## Feature request

The secret returned by the command "idpbuilder get secrets" should also include the username for argocd as this is the case for "gitea"

```
./idpbuilder get secrets
---------------------------
Name: gitea-admin-secret
Namespace: gitea
Data:
  password : xxxxxxxxxxxx
  username : giteaAdmin
---------------------------
Name: argocd-initial-admin-secret
Namespace: argocd
Data:
  password : xxxxxxxxxxxx
```

**Remark**: The name of the secret (= credential) should be more user friendly. So I suggest to rename :
- `gitea-admin-secret` to `gitea-credential`
- `argocd-initial-admin-secret` to `argocd-credential`

### PR DESCRIPTION
**Pull request by rattboi**
partially fixes #216

This only covers the gitea secret renaming, as apparently the argocd one is generated by argo on installation, and is hardcoded to `argocd-initial-admin-secret`.

See here: https://github.com/argoproj/argo-cd/blob/0f11dfb5961361807962aafc68b11426b8a47490/cmd/argocd/commands/admin/initial_password.go#L16

There's some world where we could replicate the secret into one with a name we like better, but that sounds like a recipe for confusion.

### CODE REVIEW

**Comment by cmoulliard** [comment]
Approved !

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ idpbuilder get secrets
 ```
 
 As described in the main readme file, the above command is equivalent to running:
+```bash
+kubectl get secrets -n gitea gitea-credential -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
+```
```
Comment (nabuskey) [inline CONTRIBUTING.md:67]: This is my bad. I missed copy pasta here.

```suggestion
```bash
kubectl -n argocd get secret argocd-initial-admin-secret
kubectl get secrets -n gitea gitea-admin-secret
kubectl get secrets -A -l cnoe.io/cli-secret=true
```
```

**Review by nabuskey — APPROVED** [review]
LGTM. If you could fix up my mistake I'd appreciate it :)

**Comment by elamaran11** [comment]
@rattboi @nabuskey Im getting below error while pull the latest code and deploy. Im not sure if this is related to this PR.

```
❯ k describe externalsecrets.external-secrets.io gitea-credentials -n backstage
Name:         gitea-credentials
Namespace:    backstage
Labels:       <none>
Annotations:  argocd.argoproj.io/tracking-id: backstage:external-secrets.io/ExternalSecret:backstage/gitea-credentials
API Version:  external-secrets.io/v1beta1
Kind:         ExternalSecret
Metadata:
  Creation Timestamp:  2024-04-26T18:55:27Z
  Generation:          1
  Resource Version:    3292
  UID:                 cc875094-b758-42ec-b20a-07265d00bd8e
Spec:
  Data:
    Remote Ref:
      Conversion Strategy:  Default
      Decoding Strategy:    None
      Key:                  gitea-credential
      Metadata Policy:      None
      Property:             username
    Secret Key:             GITEA_USERNAME
    Remote Ref:
      Conversion Strategy:  Default
      Decoding Strategy:    None
      Key:                  gitea-credential
      Metadata Policy:      None
      Property:             password
    Secret Key:             GITEA_PASSWORD
  Refresh Interval:         0
  Secret Store Ref:
    Kind:  ClusterSecretStore
    Name:  gitea
  Target:
    Creation Policy:  Owner
    Deletion Policy:  Retain
    Name:             gitea-credentials
Status:
  Conditions:
    Last Transition Time:  2024-04-26T18:55:27Z
    Message:               could not get secret data from provider
    Reason:                SecretSyncedError
    Status:                False
    Type:                  Ready
Events:
  Type     Reason        Age                 From              Message
  ----     ------        ----                ----              -------
  Warning  UpdateFailed  40s (x14 over 85s)  external-secrets  error retrieving secret at .data[0], key: gitea-credential, err: secrets "gitea-credential" not found
```

**Comment by rattboi** [comment]
are you using a rebuilt `idpbuilder` binary, or just the ref-implementation package against the old binary? The secret is provisioned by `idpbuilder` itself, and I would say this is a breaking change.

**Comment by nabuskey** [comment]
Yea just need to update the secret name in the example.

**Comment by elamaran11** [comment]
I think you both are right. I didnt rebuild the binary, let me try that out! Thanks!

**Comment by elamaran11** [comment]
@nabuskey Could we have a patch release for this breaking change?

**Comment by nimakaviani** [comment]
+1 on a micro release if the latest binary doesnt go with the ref implementation.

**Comment by nabuskey** [comment]
Yeah we really need to decouple the two and pin to a version

**Pull request merged**